tests/unit/serializer-test-helpers.js

```diff
@@ -0,0 +1,84 @@
+
+const appRoot = require('app-root-path');
+const { expect } = require('chai');
+const _ = require('lodash');
+
+const { openapi } = appRoot.require('utils/load-openapi');
+const { fakeId, fakeBaseUrl } = appRoot.require('tests/unit/test-data');
+
+const resourceSubsetSchema = (resourceType, resourceAttributes) => {
+  const fakeUrl = `${fakeBaseUrl}/${resourceType}s/fakeId`;
+  const schema = {
+    links: {
+      self: fakeUrl,
+    },
+    data: {
+      id: fakeId,
+      type: resourceType,
+      links: { self: fakeUrl },
+    },
+  };
+  if (resourceAttributes) {
+    schema.data.attributes = resourceAttributes;
+  }
+  return schema;
+};
+
+/**
+ * @summary Helper function for lite-testing single resource
+ * @function
```

Comment:
Could you remove `@function` since the symbol [shouldn't need it](https://jsdoc.app/tags-function.html) and remove `@summary` since you don't have a longer description already? It should look like this
```js
/**
 * Helper function for lite-testing single resource
 *
 * @param {object} ...
 */
```

Could you make this change on your other functions as well?